<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuPrincipal</class>
 <widget class="QMainWindow" name="MenuPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu Principal</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(235, 235, 235);
background-color: rgb(239, 241, 249);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnDetalle">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Historic</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Detalle Ventas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFacturas">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>334</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Facturas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>340</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Compras</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVentas">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Historic</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(249, 62, 75);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Consulta de inspección</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMClientes">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>114</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Historic</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(249, 62, 75);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Usuarios</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMProductos">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>174</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Historic</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(249, 62, 75);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Registro de inspección</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>logo-header-desktop.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSalir">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Historic</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-color: rgb(79, 79, 255);
color: rgb(255, 255, 255);
background-color: rgb(79, 79, 255);
border-color: rgb(79, 79, 255);</string>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>421</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Historic</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>rimac navidad.JPG</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>btnDetalle</zorder>
   <zorder>btnFacturas</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>btnVentas</zorder>
   <zorder>btnMClientes</zorder>
   <zorder>btnMProductos</zorder>
   <zorder>btnSalir</zorder>
   <zorder>label_4</zorder>
   <zorder>label_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>416</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
